src/harness/ascii_tables.h

```diff
@@ -0,0 +1,893 @@
+        0,          /* KEY_KP_NUMLOCK */
+        '/',        /* KEY_KP_DIVIDE */
+        '*',        /* KEY_KP_MULTIPLY */
+        '-',        /* KEY_KP_MINUS */
+        '+',        /* KEY_KP_PLUS */
+        '.',        /* KEY_KP_PERIOD */
+        0,          /* KEY_KP_EQUALS */
+        '0',        /* KEY_KP_0 */
+        '1',        /* KEY_KP_1 */
+        '2',        /* KEY_KP_2 */
+        '3',        /* KEY_KP_3 */
+        '4',        /* KEY_KP_4 */
+        '5',        /* KEY_KP_5 */
+        '6',        /* KEY_KP_6 */
+        '7',        /* KEY_KP_7 */
+        '8',        /* KEY_KP_8 */
+        '9',        /* KEY_KP_9 */
+        0,          /* KEY_F1 */
+        0,          /* KEY_F2 */
+        0,          /* KEY_F3 */
+        0,          /* KEY_F4 */
+        0,          /* KEY_F5 */
+        0,          /* KEY_F6 */
+        0,          /* KEY_F7 */
+        0,          /* KEY_F8 */
+        0,          /* KEY_F9 */
+        0,          /* KEY_F10 */
+        0,          /* KEY_F11 */
+        0,          /* KEY_F12 */
+        0,          /* KEY_PRTSCN */
+        0,          /* KEY_SCRLK */
+        0,          /* KEY_PAUSE */
+        ' ',        /* KEY_SPACE */
+    },
+};
+
+dethrace_ascii_tables carmageddon_german_ascii_tables = {
+    {
+        0,          /* KEY_SHIFT_ANY */
+        0,          /* KEY_ALT_ANY */
+        0,          /* KEY_CTRL_ANY */
+        0,          /* KEY_CTRL_ANY_2 */
+        0,          /* KEY_CAPSLOCK */
+        0,          /* KEY_RSHIFT */
+        0,          /* KEY_RALT */
+        0,          /* KEY_RCTRL */
+        0,          /* KEY_LSHIFT */
+        0,          /* KEY_LALT */
+        0,          /* KEY_LCTRL */
+        '0',        /* KEY_0 */
+        '1',        /* KEY_1 */
+        '2',        /* KEY_2 */
+        '3',        /* KEY_3 */
+        '4',        /* KEY_4 */
+        '5',        /* KEY_5 */
+        '6',        /* KEY_6 */
+        '7',        /* KEY_7 */
+        '8',        /* KEY_8 */
+        '9',        /* KEY_9 */
+        'a',        /* KEY_A */
+        'b',        /* KEY_B */
+        'c',        /* KEY_C */
+        'd',        /* KEY_D */
+        'e',        /* KEY_E */
+        'f',        /* KEY_F */
+        'g',        /* KEY_G */
+        'h',        /* KEY_H */
+        'i',        /* KEY_I */
+        'j',        /* KEY_J */
+        'k',        /* KEY_K */
+        'l',        /* KEY_L */
+        'm',        /* KEY_M */
+        'n',        /* KEY_N */
+        'o',        /* KEY_O */
+        'p',        /* KEY_P */
+        'q',        /* KEY_Q */
+        'r',        /* KEY_R */
+        's',        /* KEY_S */
+        't',        /* KEY_T */
+        'u',        /* KEY_U */
+        'v',        /* KEY_V */
+        'w',        /* KEY_W */
+        'x',        /* KEY_X */
+        'z',        /* KEY_Y */
+        'y',        /* KEY_Z */
+        '^',        /* KEY_GRAVE */
+        0xdf,       /* KEY_MINUS (ß) */
```

Comment:
I didn't verify the actual textures. So there might be cases where the character in the comments does not match the actual glyph present in the texture.